modules/kargoAnalyticsAdapter.js

```
@@ -1,96 +1,757 @@
+
+    baseAdapter.enableAnalytics.call(this, config);
+  },
+
+  /**
+   * Disable analytics and clean up
+   */
+  disableAnalytics() {
+    _config = { ...DEFAULT_CONFIG };
+    _sampled = true;
+    // Clear cache
+    Object.keys(cache.auctions).forEach(key => delete cache.auctions[key]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)